lib/spack/docs/index.rst

```diff
@@ -65,6 +65,7 @@ or refer to the full manual below.
    repositories
    command_index
    package_list
+   gpg
```

Comment:
Users will probably want to use `spack gpg init` to import the LLNL key for the default package repository. Deployments of Spack can add site-local keys for hosted packages there as well. Users generating packages will want their own key so that they may sign and trust the generated packages. Developers may want to `spack gpg trust` other developer keys for collaboration.

`spack gpg verify` may also be useful for verifying packages without installing them. The rest is management and query subcommands.